src/components/DropdownModal.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <Modal
+      id="dropdown-modal"
+      submit-text="Zugriff ändern"
+      @submit="submit"
+      @click.self="closeModal"
+  >
+
+  </Modal>
+</template>
+
+<script setup lang="ts">
+import Modal from '@/components/Modal.vue';
+import { closeModal } from '@/utils';
+import { useChannel } from '@/composables/channel';
+import {ref} from "vue";
+
+
+const props = defineProps<{
+  studentId: string;
+  channel: useChannel;
+}>();
```

Comment:
den `channel` in den props zu packen macht wenig Sinn.
Lieber wie folgt
```ts
const props = defineProps<{
  studentId: string;
}>();

const channel = useChannel();
```

Später kannste dann auf `channel` statt auf `props.channel` zugreifen